_posts/2019-12-05-elasticsearch-percolator-qualification.md

```diff
@@ -0,0 +1,198 @@
+                      "query": "("I've sent" | "I've already sent") ("check")",
+                      "fields": [
+                          "content"
+                      ],
+                      "default_operator": "and"
+                  }
+              }
+          ]
+      }
+  },
+  "tag_uuid": "2f86ad85-4c09-4ef3-bb6e-100d129018e9",
+  "tag_name": "Check sent",
+}
+```
+
+And if we search through this index, we will retrieve our newly added search query document : 
+```
+GET search-perco/_search
+{
+  "query": {"match_all": {}}
+}
+
+{
+  "took": 1,
+  "timed_out": false,
+  "_shards": {
+    "total": 5,
+    "successful": 5,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": {
+    "total": 1,
+    "max_score": 1,
+    "hits": [
+      {
+        "_index": "search-perco",
+        "_type": "_doc",
+        "_id": "1",
+        "_score": 1,
+        "_source": {
+          "query": ...
+          "tag_uuid": "2f86ad85-4c09-4ef3-bb6e-100d129018e9",
+          "tag_name": "Check sent",
+        }
+      }
+    ]
+  }
+}
+```
+
+Now it's time to percolate call documents via the percolate query : 
+```
+GET /search-perco/_search
+{
+  "_source": [
+    "tag_uuid",
+    "tag_name"
+  ],
+  "query": {
+    "percolate": {
+      "field": "query",
+      "documents": [{`
+        "unique_id": "2f86ad85-4c09-4ef3-bb6e-100d129018e7",
+        "timestamp": "2018-01-02T18:13:30+00:00",
+        "duration": 322,
+        "transcribed": true,
+        "client_name": "Acme",
+        "content": "I've already sent to you a bank check last week..."
+      }]
+    }
+  },
+  "highlight": {
+    "fields": {
+      "content": {}
+    }
+  }
+}
+```
+
+Elasticsearch providing the following response : 
+```
+{
+  "took": 37,
+  "timed_out": false,
+  "_shards": {
+    "total": 5,
+    "successful": 5,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": {
+    "total": 1,
+    "max_score": 0.8630463,
+    "hits": [
+      {
+        "_index": "search-perco",
+        "_type": "_doc",
+        "_id": "1",
+        "_score": 0.8630463,
+        "_source": {
+          "tag_name": "Check sent",
+          "tag_uuid": "2f86ad85-4c09-4ef3-bb6e-100d129018e9"
+        },
+        "fields": {
+          "_percolator_document_slot": [
+            0
+          ]
+        },
+        "highlight": {
+          "content": [
+            "<em>I've already sent</em> to you a bank <em>check</em> last week..."
+          ]
+        }
+      }
+    ]
+  }
+}
+```
+
+So here we see that our call document matched the search query tagged `Check sent`. We can the highlighter to highlight the terms that have matched from the search query documents. The field `_percolator_document_slot` is useful when we send several documents to `documents` field of the percolate query. And `max_score` and `_score` as usual give you the relevance score of matched documents. You can disable the score computing when using the percolate query using a filter context.
```

Comment:
- Missing word at `We can the highlighter`
- we send several documents to *the* `documents` field
- remove *as usual*, it's not obvious at all
- maybe link to what filter contexts are